Python/Product/PythonTools/PythonTools/Repl/BasePythonReplEvaluator.cs

```diff
@@ -1783,6 +1785,19 @@ class MemberResults {
         }
 
 #if DEV14_OR_LATER
+        internal static Version VSInteractiveVersion {
+            get {
+                if (_vsInteractiveVersion == null) {
+                    _vsInteractiveVersion = typeof(IInteractiveWindow).Assembly
+                        .GetCustomAttributes(typeof(AssemblyInformationalVersionAttribute), false)
+                        .OfType<AssemblyInformationalVersionAttribute>()
+                        .Select(a => { Version v; return Version.TryParse(a.InformationalVersion, out v) ? v : null; })
+                        .FirstOrDefault(v => v != null) ?? new Version();

```

Comment:
It's not our job to validate someone else's assembly - this is a best attempt at getting it right, and not-crashing is more important than being perfect.

If it were our own assembly (we wouldn't be doing this... but if we were...) then we'd validate it because we could fix it. There's no value here - we're in dirty hack territory already and can't really ask another team to fix their private attributes on their private assembly.